assets/dev/js/editor/document/elements/commands-internal/recreate.js

```diff
@@ -5,15 +5,27 @@ export class Recreate extends CommandInternal {
 		this.requireArgumentType( 'models', 'object', args );
 	}
 
-	apply( { models } ) {
-		Object.entries( models ).forEach( ( [ id, model ] ) => {
+	async apply( { models } ) {
+		const currentElementId = elementor.getCurrentElement()?.model?.get( 'id' );
```

Comment:
What will happen it's undefined?

Reply:
Nothing, it will not change the panel route.

Reply:
@matipojo isn't it's better to use `elementor.getPanelView().getCurrentPageView().getOption('editedElementView')` ?